internal/handlers/handler_register_duo_device.go

```diff
@@ -45,7 +45,7 @@ func SecondFactorDuoDevicesGet(duoAPI duo.API) middlewares.RequestHandler {
 		}
 
 		if result == allow {
-			ctx.Logger.Debugf("Device selection not possible for user %s, because Duo authentication was bypassed - Defaults to Auto Push", userSession.Username)
+			ctx.Logger.Debugf("Device selection not possible for user %s, because Duo authentication was bypassed - Default to Auto Push", userSession.Username)
```

Comment:
```suggestion
			ctx.Logger.Debugf("Device selection not possible for user %s, because Duo authentication was bypassed - Defaults to Auto Push", userSession.Username)
```